### PR TITLE
Feature/place layout component

### PR DESCRIPTION
**Pull request by K0sen**

### CODE REVIEW

File: frontend/src/components/place/PlacePhoto.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    props: {
+        photo: {
+            type: Object,
+            required: true
+        }
+    },
+}
+</script>
+
+<style lang="scss">
```
Comment (nemoipaha) [inline frontend/src/components/place/PlacePhoto.vue]: scoped styles

File: frontend/src/components/place/PlacePhotoList.vue [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                {id: 7, url: 'https://goo.gl/Vi38JD'},
+                {id: 8, url: 'https://goo.gl/Vi38JD'},
+                {id: 9, url: 'https://goo.gl/Vi38JD'},
+            ]
+        };
+    },
+}
+</script>
+
+<style lang="scss">
```
Comment (nemoipaha) [inline frontend/src/components/place/PlacePhotoList.vue]: scoped styles

File: frontend/src/components/place/PlaceSidebarInfo.vue [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    </aside>
+</template>
+
+<script>
+export default {
+    name: "PlaceSidebarInfo",
+}
+</script>
+
+<style lang="scss">
```
Comment (nemoipaha) [inline frontend/src/components/place/PlaceSidebarInfo.vue]: use scoped )

File: frontend/src/components/place/PlaceTopInfo.vue [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    },
+    data() {
+        return {
+            activeTab: 0
+        }
+    }
+}
+</script>
+
+<style lang="scss">
```
Comment (nemoipaha) [inline frontend/src/components/place/PlaceTopInfo.vue]: scoped )

File: frontend/src/components/place/PlaceView.vue [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+export default {
+    name: "PlaceView",
+    components: {
+        PlaceSidebarInfo,
+        PlaceTopInfo,
+    },
+}
+</script>
+
+<style>
```
Comment (nemoipaha) [inline frontend/src/components/place/PlaceView.vue]: scoped )

**Comment by K0sen** [comment]
resolve conflict & add scoped to css

File: frontend/src/App.vue [hunk 6]
```diff
@@ -1,25 +1,40 @@
 <template>
-  <div id="app">
-    <img src="./assets/logo.png">
-    <router-view/>
-  </div>
+    <div id="app">
+        <link rel="stylesheet"
```
Comment (nemoipaha) [inline frontend/src/App.vue]: Remove <link>

File: frontend/src/components/place/PlaceTopInfo.vue [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+    components: {
+        PlacePhotoList,
+    },
+    data() {
+        return {
+            activeTab: 0
+        }
+    }
+}
+</script scoped>
```
Comment (nemoipaha) [inline frontend/src/components/place/PlaceTopInfo.vue]: This will break frontend completely.

**Comment by K0sen** [comment]
fixed

**Comment by K0sen** [comment]
Resolve again)

**Pull request merged**